docs/advanced-topics/checksum-support.rst

```diff
@@ -87,6 +87,10 @@ However, in some cases the image checksum may diverge, for example, if the image
 been updated. For best results, and to avoid your Docker image being deleted because of a registry's retention policy,
 use Docker images referred to by a tagged version or digest. The verification features available may vary between execution engines.
 
+Note: Checksum gathering will only work with Docker images pushed after roughly 2016 or specifically with a Docker version after 1.10.
+This ensures that the images use the Docker V2 Schema 2 image manifest.
```

Comment:
`This ensures` doesn't sound right to me. It's not ensuring anything, it just won't fetch the checksums, but it doesn't block anything else, AFAIK.

Reply:
changed comment